ci/generate_azure_pipelines_matrices.py

```diff
@@ -368,32 +371,36 @@ def __init__(self, platform, build_jobs):
 )
 
 # Test binary patch of qmake
-linux_build_jobs.extend(
-    [
-        # New output dir is shorter than the default value; qmake could fail to
-        # locate prefix dir if the value is patched wrong
-        BuildJob(
-            "install-qt",
-            "5.12.11",
-            "linux",
-            "desktop",
-            "gcc_64",
-            "gcc_64",
-            output_dir="/t/Q",
-        ),
-        # New output dir is longer than the default value.
-        # This case is meant to work without any bugfix; if this fails, the test is setup wrong
-        BuildJob(
-            "install-qt",
-            "5.12.11",
-            "linux",
-            "desktop",
-            "gcc_64",
-            "gcc_64",
-            output_dir="/some/super/long/arbitrary/path/to" * 5,
-        ),
-    ]
-)
+for lst, os_name, version, arch, arch_dir in (
+    (linux_build_jobs, "linux", "5.12.11", "gcc_64", "gcc_64"),
+    (linux_arm64_build_jobs, "linux_arm64", "6.7.0", "linux_gcc_arm64", "gcc_arm64"),
```

Comment:
Why you test linux_arm64 with long path test case?

Reply:
I think this test is patching the `qmake` binary and so I thought there may be differences in patching an amd64 binary vs arm64. I can remove it if you think it's redundant.

Reply:
@rectalogic please see #257, #258, and #259 for context regarding these tests. IMHO, the existing tests are already sufficient for what they’re intended to test, and I don’t see a reason to add anything more here. I appreciate that you’re trying to be thorough though.

The reality is that Azure compute time isn’t cheap, and we don’t want to waste it on tests that won’t tell us much new information. Otherwise, we would be testing this property on Mac and Windows as well.